google/cloud/forseti/scanner/audit/forwarding_rule_rules_engine.py

```diff
@@ -209,40 +209,38 @@ def find_match(self, forwarding_rule):
         """
         ip_matched = forwarding_rule.ip_address == self.rules['ip_address']
 
-        scheme_matched = forwarding_rule.load_balancing_scheme == \
-            self.rules['load_balancing_scheme']
+        scheme_matched = forwarding_rule.load_balancing_scheme == self.rules[
+            'load_balancing_scheme']
 
         # only one of port or port range will be populated by the rule
         # for the port range. it is a string of form int-int
         # for the port, it will be the first and only port in the list
         ports_matched = False
         if self.rules['port_range'] and forwarding_rule.port_range:
-            ports_matched = forwarding_rule.port_range == \
-                self.rules['port_range']
+            ports_matched = forwarding_rule.port_range == self.rules[
+                'port_range']
         elif self.rules['port'] and len(forwarding_rule.ports) == 1:
-            ports_matched = int(forwarding_rule.ports[0]) == \
-                int(self.rules['port'])
+            ports_matched = int(forwarding_rule.ports[0]) == int(
+                self.rules['port'])
 
-        protocol_matched = forwarding_rule.ip_protocol == \
-            self.rules['ip_protocol']
+        protocol_matched = forwarding_rule.ip_protocol == self.rules[
+            'ip_protocol']
 
         # Checking for matching based on layer 4 protocol
         # ESP has no ports
         # TCP and UDP does have ports
         # leaving it open for new layer 4 if need be
         # default cause is no match thus a return False
         if forwarding_rule.ip_protocol == 'ESP':
-            matched = (
-                (ip_matched) and
-                (scheme_matched) and
-                (protocol_matched))
+            matched = (ip_matched and
+                       scheme_matched and
+                       protocol_matched)
         elif forwarding_rule.ip_protocol == 'TCP' \
```

Comment:
How about getting rid of the backslash continuation here?

Reply:
Nice catch, fixed based on pep8 guidelines on operators.